### PR TITLE
Create UI

### PR DESCRIPTION
**Pull request by acolombo1**
- Create app.rb file with methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain Person).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- Create main.rb with a method called main invoked at the end of the file, it does the following:
  - Presents the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, asks for parameters for the option.
  - Has a way to quit the app.

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Teacher < Person
-  def initialize(name, specialization, age = 'Unknown', parent_permission: true)
-    super(name, age, parent_permission: parent_permission)
+  def initialize(name, specialization, age = 'Unknown')
+    super(name, age, true)
```
Comment (DammyShittu) [inline teacher.rb]: - [x] I was getting an error that crashes the app when creating a teacher. You need to update the `#super` method to this:

  ```
  super(name, age, parent_permission: true)
  ```
  or update initialize and super to this:
  ```
  def initialize(name, specialization, age = 'Unknown', parent_permission: true)
    super(name, age, parent_permission: parent_permission)
  ```
  
  
  spec-1

File: README.md [hunk 2]
```diff
@@ -13,6 +13,9 @@
 - git clone https://github.com/acolombo1/School-library.git
 - cd School-library
 - Start IRB and copy the .rb files content to the command line
+## Usage
+
+- Windows: run app.exe
```
Comment (DammyShittu) [inline README.md]: - [x] Does `app.exe` or `run app.exe` really start your app? I tried them and they returned similar errors.

  I think the command you are looking for is `ruby main.rb`. Include that instead for a good documentation.

  app-exe

File: person.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 class Person < Nameable
   def initialize(name, age = 'Unknown', parent_permission: true)
```
Comment (DammyShittu) [inline person.rb:2]: - **[OPTIONAL]** I think it is `name` that should have the **'Unknown'** default value, `age`. If you wish to apply this change remember to apply the same in your student and teacher classes that inherit from this

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @acolombo1,

## Highlights
✔️ Separation of app logic from entry
✔️ Display and selection of options

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 3 above)
Comment (acolombo1) [reply person.rb:2]: I think it was the age, as it is optional, let's leave it as it is. I will take it into account if we continue with the app. Thank you!

File: teacher.rb (hunk 1 above)
Comment (acolombo1) [reply teacher.rb]: I tried creating teachers without problems. But you're right that calling the super should be done like you said. I wander in which circumstances does the error come up. Corrected, thanks.

File: README.md (hunk 2 above)
Comment (acolombo1) [reply README.md]: I messed it up with this, I'm following your recommendation

**Review by keneogu — APPROVED** [review]
Hi @acolombo1,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**